pkg/translate/output.go

```diff
@@ -79,16 +80,28 @@ func printDisconnectedBlocks(wr io.Writer, root *common.Vertex, domainVisited ma
 	return nil
 }
 
-func verifyBlockSize(domainVisited map[string]int) int {
-	blockSize := -1
-	for _, bSize := range domainVisited {
-		if blockSize == -1 {
-			blockSize = bSize
-		} else if blockSize != bSize {
-			fmt.Printf("Alert! blockSize different between NVL domains\n")
+func getBlockSize(domainVisited map[string]int, adminBlockSize string) string {
+	minDomainSize := -1
+	for _, dSize := range domainVisited {
+		if minDomainSize == -1 || minDomainSize > dSize {
+			minDomainSize = dSize
 		}
 	}
-	return blockSize
+	if adminBlockSize != "" {
+		blockSizes := strings.Split(adminBlockSize, ",")
+		planningBS, err := strconv.Atoi(blockSizes[0])
+		if err != nil {
+			fmt.Printf("Alert, strconv Atoi for admin provided blockSize %v failed with err: %v! Ignoring it\n", blockSizes[0], err)
```

Comment:
since we do not return error, could you add Prometheus metrics to monitor this issue?